video/out/vo_wayland.c

```diff
@@ -504,25 +502,31 @@ static void draw_osd(struct vo *vo)
     osd_draw(vo->osd, p->osd, pts, 0, osd_formats, draw_osd_cb, p);
 }
 
-static void flip_page(struct vo *vo)
+static void redraw(void *data, uint32_t time)
```

Comment:
It is, that comes from the frame callback originally. Though it's not really useful for us.